test/Microsoft.Dnx.Testing.CommonUtils/Directory/Dir.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```

Comment:
Yup good idea. I just checked all the files again and all usings are sorted and none of them are redundant.